### PR TITLE
fix: Pass token to the client

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

File: src/handle_event.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def handle_event(data, headers):
     else:
         client = GithubClient(
             dsn=SENTRY_GITHUB_DSN,
+            token=GH_TOKEN,
```
Comment (armenzg) [inline src/handle_event.py:25]: This is the fix. I'm glad I had Sentry alerting!

File: src/github_sdk.py [hunk 2]
```diff
@@ -21,8 +21,8 @@ class GithubClient:
     # This transform GH jobs conclusion keywords to Sentry performance status
     github_status_trace_status = {"success": "ok", "failure": "internal_error"}
 
-    def __init__(self, dsn, github_token=None) -> None:
```
Comment (armenzg) [inline src/github_sdk.py:24]: Making it mandatory.

File: tests/test_handle_event.py [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 # XXX: Fix this
 os.environ["SENTRY_GITHUB_DSN"] = "https://foo@random.ingest.sentry.io/bar"
+os.environ["GH_TOKEN"] = "irrelevant"
```
Comment (asottile-sentry) [inline tests/test_handle_event.py:8]: by the way, one way to do this is with `pytest-env`

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**